File: Hanseungwoo/Week1/problem2.py
# https://www.acmicpc.net/problem/1427

n = input()      
nums = []         

for num in n:
    nums.append(int(num))

for i in range(len(nums)):
    max_idx = i
    for j in range(i + 1, len(nums)):
        if nums[j] > nums[max_idx]:    
            max_idx = j
    nums[i], nums[max_idx] = nums[max_idx], nums[i]  

for fin in nums:
    print(fin, end='')

File: Kwonsaebom/Week4/problem2.js
// https://www.acmicpc.net/problem/1764

const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const dmNumber = Number(input[0].split(" ")[0]);

const dmPerson = input.slice(1, 1 + dmNumber);
const bmPerson = input.slice(1 + dmNumber);

const dmSet = new Set(dmPerson);
const result = [];

for (let person of bmPerson) {
  if (dmSet.has(person)) {
    result.push(person);
  }
}

result.sort();

console.log(result.length);
for (let r of result) {
  console.log(r);
}
